lib/api/core/plugins/pluginContext.js

```diff
@@ -21,44 +21,38 @@ function PluginContext (kuzzle) {
     this.errors[_.upperFirst(name)] = constructor;
   });
 
-  /*
-   lazy-loading accessors, accessible only to plugins loaded by
-   a Kuzzle Server instance
-    */
-  if (kuzzle.isServer) {
-    Object.defineProperty(this.accessors, 'passport', {
-      enumerable: true,
-      get: function () {
-        return {
-          use: kuzzle.passport.use
-        };
-      }
-    });
+  Object.defineProperty(this.accessors, 'passport', {
```

Comment:
mhh well, we don't have the `isServer` attribute anymore... 
I suggest to replace the condition by `if (kuzzle instanceof Kuzzle)`